projects/graph/graph.py

```diff
@@ -31,7 +31,23 @@ def bft(self, starting_vertex):
         Print each vertex in breadth-first order
         beginning from starting_vertex.
         """
-        pass  # TODO
+        # make a queue
```

Comment:
`BFT` is solid Bryce, looks like you've got your print statement added in a more recent commit 👍